urbanopt/geojson-modelica-translator
# bugfix from Mingzhe for chp

#### Any background context you want to provide?
CHP wasn't making any electricity
#### What does this PR accomplish?
- Disconnects CHP electricity from port A that never should have been
- Shows output from CHP electricity going to port B more explicitly (changed annotation)
- Sets a value for `preTem` that mistakenly was left out
#### How should this be manually tested?
Run the `test_chp_system.py` with pytest, run the generated model from the test in Dymola, inspect the `eleFol.PEleNet` output variable. Should look like the bottom screenshot in this comment.
#### What are the relevant tickets?
#404 
#### Screenshots (if appropriate)
District with CHP as central plant:
![chp_district](https://user-images.githubusercontent.com/19916206/128783624-1c291ec6-8840-4904-b933-6d1d2fefa69b.png)
Components of the CHP plant:
<img width="530" alt="chp_plant" src="https://user-images.githubusercontent.com/19916206/128783670-aff78634-b56f-4574-b0e6-8fe4837743cf.png">
Electrical output from the CHP plant:
![elec_output_from_chp_plant](https://user-images.githubusercontent.com/19916206/128783694-fddc3c79-aa3b-499d-a52c-3204c9dc847d.png)
